Core/Core/Providers/SocialAuth/MicrosoftAuthProvider.swift

```diff
@@ -76,10 +76,10 @@ public final class MicrosoftAuthProvider {
     }
 
     private func createClientApplication() throws -> MSALPublicClientApplication {
-        guard let config = Container.shared.resolve(ConfigProtocol.self), let appID =  config.microsoft.appID else {
+        guard let config = Container.shared.resolve(ConfigProtocol.self), let clientID =  config.microsoft.clientID else {
```

Comment:
Line Length Violation: Line should be 120 characters or less; currently it has 122 characters (line_length)